pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -0,0 +1,173 @@
+
+  $ %[1]s %[2]s ipf-alt --selector="jack=the-vipper" --virtual-ips="1.2.3.4" -o yaml --images=myrepo/myipfailover:mytag
+
+ALPHA: This command is currently being actively developed. It is intended
+       to simplify the administrative tasks of setting up a highly
+       available failover configuration.
+`
+
+func NewCmdIPFailoverConfig(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+	options := &ipf.IPFailoverConfigCmdOptions{
+		ImageTemplate:    variable.NewDefaultImageTemplate(),
+		Selector:         ipf.DefaultSelector,
+		ServicePort:      ipf.DefaultServicePort,
+		WatchPort:        ipf.DefaultWatchPort,
+		NetworkInterface: ipf.DefaultInterface,
+		Replicas:         1,
+	}
+
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("%s [<name>]", name),
+		Short: shortDesc,
+		Long:  fmt.Sprintf(description, parentName, name),
+		Run: func(cmd *cobra.Command, args []string) {
+			processCommand(f, options, cmd, args, out)
+		},
+	}
+
+	cmd.Flags().StringVar(&options.Type, "type", ipf.DefaultType, "The type of IP failover configurator to use.")
+	cmd.Flags().StringVar(&options.ImageTemplate.Format, "images", options.ImageTemplate.Format, "The image to base this IP failover configurator on - ${component} will be replaced based on --type.")
+	cmd.Flags().BoolVar(&options.ImageTemplate.Latest, "latest-images", options.ImageTemplate.Latest, "If true, attempt to use the latest images instead of the current release")
+	cmd.Flags().StringVarP(&options.Selector, "selector", "l", options.Selector, "Selector (label query) to filter nodes on.")
+	cmd.Flags().StringVar(&options.Credentials, "credentials", "", "Path to a .kubeconfig file that will contain the credentials the router should use to contact the master.")
+
+	cmd.Flags().BoolVar(&options.Create, "create", options.Create, "Create the configuration if it does not exist.")
+
+	cmd.Flags().StringVar(&options.VirtualIPs, "virtual-ips", "", "A set of virtual IP ranges and/or addresses that the routers bind and serve on and provide IP failover capability for.")
+	cmd.Flags().StringVarP(&options.NetworkInterface, "interface", "i", "", "Network interface bound by VRRP to use for the set of virtual IP ranges/addresses specified.")
+
+	// unicastHelp := `Send VRRP adverts using unicast instead of over the VRRP multicast group. This is useful in environments where multicast is not supported. Use with caution as this can get slow if the list of peers is large - it is recommended running this with the label option to select a set of nodes.`
+	// cmd.Flags().StringVarP(&options.UseUnicast, "unicast", "u", options.UseUnicast, unicastHelp)
+
+	cmd.Flags().IntVarP(&options.WatchPort, "watch-port", "w", ipf.DefaultWatchPort, "Port to monitor or watch for resource availability.")
+	cmd.Flags().IntVarP(&options.Replicas, "replicas", "r", options.Replicas, "The replication factor of this IP failover configuration; commonly 2 when high availability is desired.")
```

Comment:
I don't think this comment is accurate - you probably should be telling people to set this number equal to the number of nodes for selector